src/modules/patternReplace/patternReplace.module.ts

```diff
@@ -32,10 +52,34 @@ export const patternReplace: BotModule = {
       const hasModification = message.content !== modifiedContent;
       if (!hasModification) return;
 
-      const newMessage = [`<@${message.author.id}>`, modifiedContent].join('\n');
+      const newMessageContent = [`<@${message.author.id}>`, modifiedContent].join('\n');
+
+      const newMessage = await message.channel.send(newMessageContent);
+
+      await newMessage.react('❌');
+      await message.react('❌');
```

Comment:
why did you keep this message ? look like it need to be delete and keep only newMessage

Reply:
I'm reworking the whole thing atm (it's a draft PR). We talk about this feature in the discord, and we said that it's better that the bot reply to the original message with new vxtwitter preview and only if it's a video